src/run.rs

```diff
@@ -670,6 +671,32 @@ fn vacuum(
         });
     }
 
+    // If the `--min-age` argument is provided, we need to filter out images
+    // which are newer than the provided duration.
+    if let Some(duration) = min_age {
+        match (SystemTime::now() - *duration).duration_since(UNIX_EPOCH) {
+            Ok(time_stamp) => {
+                sorted_image_nodes.retain(|(_, image_node)| {
+                    if image_node.last_used_since_epoch > time_stamp {
+                        for repository_tag in &image_node.image_record.repository_tags {
```

Comment:
I think it's a slightly nicer UX to just print the message once, with one of the tags if there happen to be multiple. Does the suggestion from my last comment work?

Reply:
Kinda, what do you think about a solution where i log the repository only or `{repository}:*` where `*` represents all tags?

Reply:
Hm, I'm not sure why you would only want the repository and not the tag. That's not enough information to uniquely find the image.

Reply:
In that case, it wouldn't work. The issue is that images are retained based on the `last_used_since_epoch` which exists on the `ImageNode`. The `ImageNode` has an `ImageRecord` which holds a vector of one repository and it's tags. For this reason the ImageNode is not directly linked to one specific image but rather all tagged images from the given repository.

Reply:
I believe you are incorrect. An `ImageRecord` has a vector of N (repository, tag) pairs. Not a single repository and N tags.

For example, an image might be tagged as both (`ubuntu`, `24.04`) and (`ubuntu`, `latest`).

Reply:
So I'm proposing that we arbitrarily use the first (repository, tag) pair for the log message. There is guaranteed to be at least one (see the `at_least_one_repository_tag` note in the code), although for untagged images it's not very useful (both the repository and tag are just the string `"<none>"` in that case). But that's probably good enough for now?

Reply:
Alternatively, we could just log the image ID—that might be better anyway? You can get it by changing `sorted_image_nodes.retain(|(_, image_node)| { ...` to `sorted_image_nodes.retain(|(image_id, image_node)| { ...`.

Reply:
Oh okay, my bad. 

I believe it is still not possible to provide the debug log with the repository and tag, an alternative would be to provide the image id, which is easily accessible on the `ImageNode`, what do you think about that solution?

Reply:
That works! I'll merge the PR after the CI passes.

(Still, I'm not sure why https://github.com/stepchowfun/docuum/pull/303#discussion_r1588805524 wouldn't work. But I'm happy with logging the image ID instead.)